tests/net/socket/af_packet_ipproto_raw/prj.conf

```diff
@@ -0,0 +1,31 @@
+# Networking config
+CONFIG_NETWORKING=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_IPV6=n
+CONFIG_NET_UDP=n
+CONFIG_NET_TCP=n
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_POSIX_NAMES=y
+CONFIG_NET_SOCKETS_PACKET=y
+CONFIG_POSIX_MAX_FDS=8
+CONFIG_NET_SOCKETS_LOG_LEVEL_DBG=y
+CONFIG_NET_CORE_LOG_LEVEL_DBG=y
+
+# Network driver config
+CONFIG_NET_L2_ETHERNET=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+
+CONFIG_NET_L2_DUMMY=y
+CONFIG_NET_IF_MAX_IPV4_COUNT=4
+
+# Network address config
+CONFIG_NET_CONFIG_SETTINGS=y
+#CONFIG_NET_CONFIG_NEED_IPV4=y
+#CONFIG_NET_CONFIG_NEED_IPV6=n
```

Comment:
Please just remove this instead of commenting out. I think we don't need NET_CONFIG_SETTINGS either.